internal/manifests/collector/adapters/config_to_ports.go

```diff
@@ -71,36 +71,35 @@ func ConfigToComponentPorts(logger logr.Logger, cType ComponentType, config map[
 	}
 
 	ports := []corev1.ServicePort{}
-	for key, val := range components {
```

Comment:
this function doesn't entirely work yet. I need to refactor this to be far simpler